### ISSUE TITLE
AvgPool2d in Downsample

### ISSUE THREAD

**Issue by JJZHK** [Pritam-N/ParNet#2]
In Downsample, I think it should be torch.nn.AvgPool2d(kernel_size=(3, 3), stride=(2, 2), padding=(1, 1)), not torch.nn.AvgPool2d(kernel_size=(2,2)). otherwise x & y have wrong size, and x + y will throw exception.

**Comment by Pritam-N** [comment]
@JJZHK  Based on the architecture in the paper, it has avgpool (2,2) and the code is working.
